modules/document_repository/README.md

```diff
@@ -0,0 +1,46 @@
+# Document Repository
+
+## Purpose
+
+The Document Repository is a tool that provides a centralized location for non-candidate documents relevant to the study. For example, copies of non-filled instruments, publications, manual of operations.
+
+## Intended Users
+
+ LORIS users can view, download/upload files, edit information about files, and delete files given appropriate permissions.
+This module is used primarily by study coordinators/administrators who share study protocols and manuals, and by data entry staff who consult these documents.  
```

Comment:
suggestion: "This module is used primarily by study coordinators/administrators who share study protocols and manuals, as well as data entry staff who consult these documents."